src/main/java/de/oliver/fancynpcs/commands/npc/CreateCMD.java

```diff
@@ -23,17 +24,22 @@ public List<String> tabcompletion(@NotNull Player player, @Nullable Npc npc, @No
     }
 
     @Override
-    public boolean run(@NotNull Player player, @Nullable Npc npc, @NotNull String[] args) {
+    public boolean run(@NotNull CommandSender receiver, @Nullable Npc npc, @NotNull String[] args) {
+        if (!(receiver instanceof Player player)) {
+            MessageHelper.error(receiver, lang.get("npc-command.only_player"));
```

Comment:
Should be "only-players" instead of "npc-command.only_player"

Reply:
I didn't change the message name at all from your previous code

Reply:
Oh, you're right, but it should "only-players" anyways. Feel free to do that, otherwise I can also do it

Reply:
For sure